ClientApp/KachnaOnline/src/app/shared/services/authentication.service.ts

```diff
@@ -417,8 +432,25 @@ export class AuthenticationService {
         this.storeUserDataToStorage();
         this.toastr.success("Změna souhlasu s gamifikací proběhla úspěšně.", "Změna uživatelských údajů");
       }).catch((error: any) => {
-      throwError(error);
-      this.toastr.error("Změna souhlasu s gamifikací selhala.", "Změna uživatelských údajů");
+        throwError(error);
+        this.toastr.error("Změna souhlasu s gamifikací selhala.", "Změna uživatelských údajů");
+    });
+  }
+
+  changeCardCode(cardCode: string) {
+    this.http.put<KisLoggedInUserInformation>(`${environment.kisApiUrl}/users/me/rfid`,
+      JSON.stringify(cardCode.toUpperCase()), {headers: new HttpHeaders({'Content-Type': 'application/json'})}).toPromise()
+      .then((res: KisLoggedInUserInformation) => {
+        this.kisLoggedInUserInformation = res;
+        localStorage.setItem(environment.kisLoggedInUserInformationStorageName, JSON.stringify(this.kisLoggedInUserInformation));
+
+        this.assignDataFromKisUserInformation();
+        this.assignDataFromLocalTokenContent();
+        this.storeUserDataToStorage();
+        this.toastr.success("Nastavení přiřazovacího kódu karty proběhlo úspěšně.", "Změna uživatelských údajů");
```

Comment:
Change the message to "Nastavení kódu karty proběhlo úspěšně."

Reply:
Done.